Makefile

```diff
@@ -1,13 +1,10 @@
-imports = \
-	@testable import GenTests;
-
 xcodeproj:
 	PF_DEVELOP=1 swift run xcodegen
 
 linux-main:
 	swift test --generate-linuxmain
 
-test-linux: linux-main
+test-linux:
```

Comment:
I still think we want to keep this, no? Unless we switch to auto discovery?

Reply:
this command currently only works when run locally since that `linux-main` can run on macOS and the rest runs docker and linux.

but on CI we can only run linux stuff. we just have to remember to call `make linux-main` every once in a while locally until we get the auto discovery stuff in.

Reply:
Oh right keep forgetting CI needs to run this line.